### ISSUE TITLE
[Bug] Fireworks from end villages are unstackable with normal crafted fireworks

### ISSUE THREAD

**Issue by ayellowlizard** [quiqueck/BetterEnd#309]
### What happened?

What happened:
Fireworks from end villages not stacking with the fireworks I crafted myself
What I expected to happen:
Fireworks from end villages stacking with the fireworks I crafted

Seems like you guys forgot to spawn the fireworks in with flight duration, while normal fireworks always have a flight duration. Should be fixed pretty easily by adding the flight duration in.

![image](https://github.com/quiqueck/BetterEnd/assets/89428651/a4997ef4-6855-4b32-a1a6-6f267a4a719e)
![image](https://github.com/quiqueck/BetterEnd/assets/89428651/59d49e8c-9ea5-4d55-802e-58ba3b39cd3a)

### BetterEnd

better-end-4.0.10

### BCLib

bclib-3.0.13

### Fabric API

fabric-api-0.87.0+1.20.1

### Fabric Loader

0.14.22

### Minecraft

1.20.1

### Other Mods

```shell
Mods:
  [📁] _bclib_deactivated (folder)
  [✔️] animatica-0.6+1.20.jar
  [✔️] architectury-9.1.12-fabric.jar
  [✔️] bclib-3.0.13.jar
  [✔️] better-end-4.0.10.jar
  [✔️] better-nether-9.0.9.jar
  [✔️] BetterPingDisplay-Fabric-1.20.1-1.1.1.jar
  [✔️] boosted-brightness-2.2.0+1.20.1.jar
  [✔️] capes-1.5.2+1.20-fabric.jar
  [✔️] cardinal-components-api-5.2.2.jar
  [✔️] chat_heads-0.10.20-fabric-1.20.jar
  [✔️] cloth-config-11.1.106-fabric.jar
  [✔️] Cobblemon-fabric-1.4.0+1.20.1.jar
  [✔️] Companion-1.20-fabric-5.1.0.jar
  [✔️] completeconfig-2.5.0.jar
  [✔️] cristellib-1.1.3-fabric.jar
  [✔️] EasyMagic-v8.0.1-1.20.1-Fabric.jar
  [✔️] enchantments-plus-fabric-1.1.11+1.20.1.jar
  [✔️] entityculling-fabric-1.6.2-mc1.20.1.jar
  [✔️] ExplorersCompass-1.20.1-2.2.3-fabric.jar
  [✔️] fabric-api-0.87.0+1.20.1.jar
  [✔️] fabric-language-kotlin-1.10.10+kotlin.1.9.10.jar
  [✔️] fabrictailor-2.1.2.jar
  [✔️] ferritecore-6.0.0-fabric(1).jar
  [✔️] ForgeConfigAPIPort-v8.0.0-1.20.1-Fabric.jar
  [✔️] forgottengraves-1.20.1-3.2.1.jar
  [✔️] geckolib-fabric-1.20.1-4.2.2.jar
  [✔️] grasskiss-1.20-fabric0.83.0-0.5.0.jar
  [✔️] GuardiansGalore-1.20.1-3.1.jar
  [✔️] iris-mc1.20.1-1.6.8.jar
  [✔️] lambdynamiclights-2.3.1+1.20.1.jar
  [✔️] litematica-fabric-1.20.1-0.15.3.jar
  [✔️] malilib-fabric-1.20.1-0.16.3.jar
  [✔️] memoryleakfix-fabric-1.17+-1.1.2.jar
  [✔️] modmenu-7.2.1.jar
  [✔️] mousewheelie-1.12.2+mc1.20.1.jar
  [✔️] NaturesCompass-1.20.1-2.2.3-fabric.jar
  [✔️] NoChatReports-FABRIC-1.20.1-v2.2.2.jar
  [✔️] Origins-1.20.1-1.10.0.jar
  [✔️] Origins-Classes-1.20-1.7.0.jar
  [✔️] origins-plus-plus-2.2.jar
  [✔️] owo-lib-0.11.0+1.20.jar
  [✔️] Patchouli-1.20.1-81-FABRIC(1).jar
  [✔️] Pehkui-3.7.8+1.14.4-1.20.1.jar
  [✔️] PuzzlesLib-v8.0.15-1.20.1-Fabric.jar
  [✔️] repurposed_structures-7.1.8+1.20.1-quilt.jar
  [✔️] resolution-control-plus-1.20-3.0.0.jar
  [✔️] RoughlyEnoughItems-12.0.665(2).jar
  [✔️] sneaktweak-1.10.0.jar
  [✔️] sodium-fabric-mc1.20.1-0.5.2.jar
  [✔️] starlight-1.1.2+fabric.dbc156f.jar
  [✔️] TerraBlender-fabric-1.20.1-3.0.0.169.jar
  [✔️] Terralith_1.20.1_v2.4.5.jar
  [✔️] travelers-backpack-1.20.1-9.1.3.jar
  [✔️] victus-0.1.14+1.20(1).jar
  [✔️] WilderWild-2.1.3-Fabric+1.20.1.jar
  [✔️] WorldEditCUI-1.20+01.jar
  [✔️] worldedit-mod-7.2.15(1).jar
  [✔️] wraith-waystones-3.1.3+mc1.20.jar
  [✔️] wwoo-3.2.1-fabric.jar
  [✔️] Xaeros_Minimap_23.8.2_Fabric_1.20.jar
  [✔️] XaerosWorldMap_1.34.1_Fabric_1.20.jar
  [✔️] yet-another-config-lib-fabric-3.1.1+1.20.jar
  [✔️] Zoomify-2.11.2.jar

Params:
  --username A_Yellow_Lizard --version 1.20.1 --gameDir C:/Misc/Games/MultiMC/instances/HDIA/.minecraft --assetsDir C:/Misc/Games/MultiMC/assets --assetIndex 5 --uuid <PROFILE ID> --accessToken <ACCESS TOKEN> --userType msa --versionType release

Window size: 854 x 480

Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms512m, -Xmx4480m, -Duser.language=en]
```

**Comment by quiqueck** [comment]
Thanks for the report. This will be fixed (for unopened chests) in the next release.

**Comment by quiqueck** [comment]
Just released the (hopefully) fixed version (4.0.11)
